protected/humhub/migrations/m220207_183901_add_message_column_to_notification_table.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use yii\db\Migration;
+
+/**
+ * Add `message` column to `notification` table
+ */
+class m220207_183901_add_message_column_to_notification_table extends Migration
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function safeUp()
+    {
+        $this->addColumn('notification', 'message', $this->text());
```

Comment:
@s-tyshchenko  Instead of adapting the BaseNotification for this case (`message`) only, it would be better to create a more generic way to pass additional payload to a notification.

How about adding a field `payload` or `data` for example. This could always be an array with data and could be automatically JSON de/encoded by `BaseNotification`.

e.g. `$this->payload['message']`